bbb-libreoffice/docker/Dockerfile

```diff
@@ -1,13 +1,3 @@
-FROM openjdk:17-slim-bullseye
-ENV DEBIAN_FRONTEND noninteractive
-
-RUN echo "deb http://deb.debian.org/debian bullseye-backports main" >> /etc/apt/sources.list
-RUN apt update && apt -y install locales-all fontconfig libxt6 libxrender1
-RUN apt update && apt -y install libreoffice \
-  && rm -f \
-  /usr/share/java/ant-apache-log4j-1.10.9.jar \
```

Comment:
Hello @gustavotrott - please check what is the version of log4j that's installed in this new implementation
I believe these removals was to ensure the vulnerable version would not be there.

Reply:
The log4j files is no longer present in this image!

![image](https://user-images.githubusercontent.com/5660191/193045664-cc18ed40-0979-4a94-8f28-448b792ab149.png)

Reply:
great!